### PR TITLE
Feature/item package

### PR DESCRIPTION
**Pull request by Sebsta13**
### All changes meet the following requirements
- [x] Changelog entry was added
- [x] Changes have been tested
- [x] Plugin can be built

@plentymarkets/ceres-io

### CODE REVIEW

File: resources/js/src/app/components/item/ItemBundle.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    {
+        getBundleInnerText(item)
+		{
+            item.variation.bundleType = null;
+
+            return item;
+        }
+    },
+
+    computed:
```
Comment (cjohannsen) [inline resources/js/src/app/components/item/ItemBundle.js]: @Strean1337 place computed below data

**Review by cjohannsen — APPROVED** [review]

File: resources/lang/en/Template.properties [hunk 2]
```diff
@@ -337,6 +340,7 @@ notificationsCalculateShippingFailed = "Error while calculating shipping costs.
 notificationsCheckPassword="The passwords do not match."
 notificationsInvalidResetPasswordUrl="The entered URL is invalid or expired."
 notificationsItemNotAdded = "The item could not be added."
+notificationsItemBundleSplitted = "Item bundle splitted"
```
Comment (fmutschler) [inline resources/lang/en/Template.properties:343]: "Item bundle has been split."

File: resources/lang/en/Template.properties [hunk 3]
```diff
@@ -223,6 +223,9 @@ homepageShowAll = "Show all"
 itemApply = "Apply"
 itemAvailabilityAverageDays_asc = "Availability ⬆"
 itemAvailabilityAverageDays_desc = "Availability ⬇"
+itemBundle = "Item package"
+itemBundleName = "[Bundle] :itemName"
+itemBundleContent = "Item package content:"
```
Comment (fmutschler) [inline resources/lang/en/Template.properties:228]: "Item bundle content:"

File: resources/lang/de/Template.properties [hunk 4]
```diff
@@ -337,6 +340,7 @@ notificationsCalculateShippingFailed = "Bei der Versandkostenberechnung kam es z
 notificationsCheckPassword="Die Passwörter stimmen nicht überein."
 notificationsInvalidResetPasswordUrl="Die angegebene URL ist ungültig oder abgelaufen."
 notificationsItemNotAdded = "Der Artikel konnte nicht hinzugefügt werden"
+notificationsItemBundleSplitted = "Artikelpaket wurde aufgeteilt"
```
Comment (fmutschler) [inline resources/lang/de/Template.properties:343]: "Artikelpaket wurde aufgeteilt."

File: resources/lang/en/Template.properties [hunk 5]
```diff
@@ -223,6 +223,9 @@ homepageShowAll = "Show all"
 itemApply = "Apply"
 itemAvailabilityAverageDays_asc = "Availability ⬆"
 itemAvailabilityAverageDays_desc = "Availability ⬇"
+itemBundle = "Item package"
```
Comment (fmutschler) [inline resources/lang/en/Template.properties:226]: "Item bundle"

**Pull request merged**